src/main/resources/db/changelog/20230220150433-drop-act-key-env-tables.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+        http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.1.xsd">
```

Comment:
We're currently on liquibase 4.19, so let's link to the `dbchangelog-4.19.xsd` and make it a habit to use that from now on. It will help with validation if/whenever we might use newer liquibase xml tags